tensorflow/python/kernel_tests/reader_ops_test.py

```diff
@@ -858,5 +858,40 @@ def _RunSessionAndSave(sess, args, output):
     output.append(sess.run(args))
 
 
+class LMDBReaderTest(test.TestCase):
+
+  def setUp(self):
+    super(LMDBReaderTest, self).setUp()
+
+  def testReadFromFile(self):
+    with self.test_session() as sess:
+      reader = io_ops.LMDBReader(name="test_read_from_file")
+      path = os.path.join("tensorflow", "core", "lib", "lmdb", "testdata",
+                          "data.mdb")
+      queue = data_flow_ops.FIFOQueue(99, [dtypes.string], shapes=())
+      key, value = reader.read(queue)
+
+      queue.enqueue([path]).run()
+      queue.close().run()
+      for i in range(10):
+        k, v = sess.run([key, value])
+        self.assertEqual(k, str(i))
+        self.assertEqual(v, str(chr(ord('a') + i)))
+
```

Comment:
Please test that you get an assertion when reading past the end, see, e.g., https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/kernel_tests/reader_ops_test.py#L483